user-app-test/src/modules/auth/dto/sign-in.dto.ts

```diff
@@ -0,0 +1,9 @@
+import { IsEmail, IsString } from 'class-validator';
+
+export class SignInDto {
+  @IsEmail()
+  email: string;
+
+  @IsString()
```

Comment:
чуточку больше валидаций хотелось бы, потому что вот это прокатит в данном случае: ''
поэтому можно добавить:
```suggestion
  @IsRegExp(new RegExp(alphanumberic + special char))
  @IsString()
  @IsNotEmpty()
```

Reply:
понял, добавлю